### PR TITLE
Pill component

### PR DESCRIPTION
**Pull request by chriszarate**
(Also exporting a few useful components.)

Screen Shot 2020-09-14 at 2 38 02 PM

### CODE REVIEW

**Review by k-vosswinkel — APPROVED** [review]
💊

File: src/components/Pill/Pill.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Pill from './Pill';
+
+const defaultProps = {
+	children: 'Test content',
```
Comment (k-vosswinkel) [inline src/components/Pill/Pill.test.js:6]:
```suggestion
	text: 'Test content',
```

File: src/components/Pill/Pill.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@import '~@quartz/styles/scss/color-scheme';
+@import '~@quartz/styles/scss/fonts';
+@import '~@quartz/styles/scss/media-queries';
+
+.container {
+	@include font-maison-extended-700-1;
+
+	background-color: $color-border-decorative;
+	border-radius: 0.75em;
```
Comment (jshakes) [inline src/components/Pill/Pill.scss:9]: should this use the `$size-border-radius` token?

File: src/components/Pill/Pill.test.js (hunk 1 above)
Comment (chriszarate) [reply src/components/Pill/Pill.test.js:6]: oopsie ty

File: src/components/Pill/Pill.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	text,
+} ) {
+	return <span className={styles.container}>{text}</span>;
+}
+
+Pill.propTypes = {
+	/**
+	 * String to be rendered inside the pill.
+	 */
+	text: PropTypes.string.isRequired,
```
[the remark this answers is not in the document]
Comment (chriszarate) [reply src/components/Pill/Pill.jsx:15]: whatchu gonna put in there?

File: src/components/Pill/Pill.scss (hunk 2 above)
Comment (chriszarate) [reply src/components/Pill/Pill.scss:9]: that doesn't match design, but i'll query the daniels-that-be

File: src/components/Pill/Pill.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (jshakes) [reply src/components/Pill/Pill.jsx:15]: text! but using children feels more react-y and readable. we can always throw a proptype warning if someone tries to use JSX if we feel strongly about that

[the remark this answers is not in the document]
Comment (chriszarate) [reply src/components/Pill/Pill.jsx:15]: > we can always throw a proptype warning if someone tries to use JSX if we feel strongly about that

but that's like ... what `string` and `children` do. but i get ya, more about the dev experience

**Review by k-vosswinkel — APPROVED** [review]
🎉

**Pull request merged**